io.github.mpobaschnig.Vaults.json

```diff
@@ -21,18 +21,33 @@
     ],
     "build-options" : {
         "append-path" : "/usr/lib/sdk/rust-stable/bin",
-        "build-args" : [
-            "--share=network"
-        ],
-        "test-args" : [
-            "--socket=x11",
-            "--share=network"
-        ],
         "env" : {
             "RUST_BACKTRACE" : "1"
         }
     },
     "modules" : [
+        {
+            "name" : "libsass",
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/lazka/libsass.git",
+                    "commit" : "302397c0c8ae2d7ab02f45ea461c2c3d768f248e"
+                }
+            ]
+        },
+        {
+            "name" : "sassc",
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/lazka/sassc.git",
+                    "commit" : "82803377c33247265d779af034eceb5949e78354"
+                }
+            ]
+        },
```

Comment:
Both these are build dependencies of libadwaita, you should add `"cleanup": ["*"]` to both of them to remove them completely post-build

Reply:
Added.